Tests/Resources/Fixtures/loader_config/valid7.yml

```diff
@@ -0,0 +1,12 @@
+stdClass:
+    definitions: 
+        - 
+            uri_schema: /forum/{category}/{post_title}/edit
+            defaults:
+                _type: edit
+        -
```

Comment:
The names only matter when you are extending an existing definition, i.e. they enable you to override previously defined definitions. So 0, 1, etc are perfectly valid.